install.sh

```diff
@@ -32,11 +32,28 @@ if [ -e "$HOME/$BACKUP_FILE" ]; then
     done
 fi
 
-test -w "$HOME/$CONFIG_FILE" &&
-  cp -a "$HOME/$CONFIG_FILE" "$HOME/$CONFIG_FILE.bak" &&
-  echo -e "\033[0;32mYour original $CONFIG_FILE has been backed up to $CONFIG_FILE.bak\033[0m"
+while true
+do
+  read -e -n 1 -r -p "Would you like to keep your $CONFIG_FILE and append bash-it templates at the end? [y/N] " choice
+    case $choice in
+      [yY])
+	    (sed "s|{{BASH_IT}}|$BASH_IT|" "$BASH_IT/template/bash_profile.template.bash" || tail -n +2) >> "$HOME/$CONFIG_FILE"
```

Comment:
Yeah, you are totally right. Sorry, but I'm an Android dev and sometimes i mix the syntax... ;)